packages/dashboard/src/components/queryEditor/advancedSearch/model/property.ts

```diff
@@ -0,0 +1,5 @@
+export interface Property {
+  propertyId: string;
+  propertyName: string;
+  displayName: string;
+}
```

Comment:
Is it possible to pull this type from the SDK?

Reply:
I don't think there's a single interface that contains all the fields we want to display in the table. My thinking was this interface represents the fields we're displaying in the table and so any response we would want to convert into this form. 

Specifically for Knowledge Graph responses, when we select different fields in the query (for example `Select e.entityName as assetName` then the response does not have Typescript types for the fields, so this interface represents the form we will convert the response to in order to display in the table.